package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "devDependencies": {
+    "cross-env": "^7.0.3",
+    "cssnano": "^7.0.6",
+    "nodemon": "^3.1.7",
+    "postcss": "^8.4.49",
+    "sass": "^1.81.0"
+  },
+  "scripts": {
+    "build": "cross-env NODE_ENV=production node build.js",
+    "watch": "nodemon --exec node build.js --ignore js/jsoncity.js --ignore css/style.css --ext js,scss"
```

Comment:
I was really trying to use the very most simplistic thing ever but it took chatbot like an hour to figure out something simple. Was pushing webpacks on me and I'm like bro nonono

Reply:
The problem with webpack is in javascript :rage4:
Rust is quite trendy for addressing similar new issues, but Vite is is Go I think..
Personally, I just use Bun runtime for smaller stuff, it just works by default :smiling_imp::dash: